apps/win32/.npmignore

```diff
@@ -0,0 +1,4 @@
+yarn-error.log
+
+# Build targets
+!dist
```

Comment:
Do you want to ignore the `src` files? You can try running `npm pack --dry-run` to see what files would be included in the npm files. Then add the files that you don't want to pack to this file.

Reply:
I'm not sure if I do. Is there a standard here or is it just preference? When I try to ignore `src` it doesn't ignore the `src` folder in the root of the package but instead the `src` folders under `lib` and `lib-commonjs`